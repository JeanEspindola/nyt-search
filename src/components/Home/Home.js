/*
* Home Presentation Component
*
* Displays the basic components for searching and list of articles.
*
* */
import React, { Fragment } from 'react';
import SearchInput from '../Search-Input/SearchInputContainer';
import ArticleList from '../Article-List/ArticleListContainer';

const Home = () => (
  <Fragment>
    <SearchInput />
    <ArticleList />
  </Fragment>
);

export default Home;
