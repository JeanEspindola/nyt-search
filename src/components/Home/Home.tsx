/*
* Home Presentation Component
*
* Displays the basic components for searching and list of articles.
*
* */
import SearchInput from '../Search-Input/SearchInput'
import ArticleList from '../Article-List/ArticleList'

const Home = () => (
  <>
    <SearchInput />
    <ArticleList />
  </>
);

export default Home;
