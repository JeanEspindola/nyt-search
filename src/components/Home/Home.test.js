import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Home from './Home';

describe('Home', () => {
  let Component;

  beforeEach(() => {
    Component = shallow(<Home />);
  });

  it('renders Home without crashing', () => {
    expect(Component.length).toBeTruthy();
    expect(shallowToJson(Component)).toMatchSnapshot();
  });
});
