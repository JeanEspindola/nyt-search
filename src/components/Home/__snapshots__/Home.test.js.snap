// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders Home without crashing 1`] = `
<div
  className="Home"
>
  <Connect(SearchInput) />
  <Connect(ArticleList) />
</div>
`;
